stl/inc/yvals_core.h

```diff
@@ -67,6 +67,7 @@
 // P1165R1 Consistently Propagating Stateful Allocators In basic_string's operator+()
 // P1902R1 Missing Feature-Test Macros 2017-2019
 // P2401R0 Conditional noexcept For exchange()
+// P2419R2 Clarify Handling Of Encodings In Localized Formatting Of chrono Types

```

Comment:
This is specific to the behavior of C++20 chrono formatting. Therefore it should be listed under `_HAS_CXX20`, not the truly unconditional (i.e. C++14) section.